tee-worker/docker/lit-di-evm-identity-test.yml

```diff
@@ -17,7 +17,10 @@ services:
                 condition: service_healthy
         networks:
             - litentry-test-network
-        entrypoint: "bash -c '/usr/local/worker-cli/lit_ts_test.sh test-di-evm-identity 2>&1' "
+        entrypoint: |
+            /usr/local/worker-cli/lit_ts_api_package_build.sh -p 9912 -u ws://litentry-node -W http://litentry-node -V wss://litentry-worker-1 -A 2011 -C /usr/local/bin/litentry-cli 2>&1
+            && \
+            /usr/local/worker-cli/lit_ts_test.sh test-resuming-worker 2>&1
```

Comment:
Hm, maybe it'd be cleaner to wrap it into the `lit_ts_test.sh` script itself? ^_^

Reply:
Thanks, valid comment. Initially, I thought the same and later came to the conclusion that running tests and preparing setup for them might be two different intentions. Like if there is already generated metadata. Though I agree that there is a space for refactoring